main/src/ca/uwaterloo/flix/language/phase/Reader.scala

```diff
@@ -16,15 +16,14 @@
 
```

Comment:
Why these unrelated changes?

Reply:
There were unused imports, so I figured I may as well clean that up but it can be reverted no problem :+1:

Reply:
OK, thats fine.